sourse/user_class_unittests.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from user_class import User
```

Comment:
You need to move the tests to a separate directory

Reply:
ok!